docs/apis-tools/operate-api/overview.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: operate-api-overview
 title: Overview
-slug: /operate-api/overview
+slug: /apis-tools/operate-api/overview
```

Comment:
@christinaausley this (and the related change in https://github.com/camunda/camunda-platform-docs/pull/1811/commits/260ae8e152f47fd01b36c74a06c8b6e745e2a639#diff-72c9352882d3321d62f2c9c5a6d36fc2e189c4d5cb4e940cb0ef5ec1a071e729) should fix the rest of the link errors.

This slug should include the whole path after `docs` and the optional version, so that the URL reflects the same hierarchy as in the left nav. 

The actual error was happening because of the sidebars files -- they were looking for `/docs/next/apis-tools/operate-api/overview`, which didn't exist, because the files in this folder didn't specify the `apis-tools` part.